WTWSMS/decisions/realm_decisions.txt

```diff
@@ -1271,13 +1271,16 @@ decisions = {
 		potential = {
 			has_dlc = "Sons of Abraham"
 			is_playable = yes
-			religion_group = jewish_group
+			OR = {
+				religion_group = jewish_group
+				religion = ebionites
+			}
 			k_israel = {
 				has_holder = no
 			}
 		}
 		allow = {
-			completely_controls = k_oriens
+			completely_controls = d_iudea # K_ORIENS IS TOO BIG, THIS IS WHAT ISRAEL HAD IN RL
```

Comment:
@IhateTrains 

I'll accept that if we can get it closer to the borders in the historical maps of Israel. What about the two counties directly above "D_IUDEA", and then the other county directly to its right?